lazyflow/operators/ioOperators/opExportSlot.py

```diff
@@ -236,40 +240,46 @@ def run_export(self):
         else:
             mkdir_p( PathComponents(self.ExportPath.value).externalDirectory )
             export_func()
-    
-    def _export_hdf5(self, compress=False):
+
+    def _export_h5n5(self, compress=False):
         self.progressSignal( 0 )
 
-        # Create and open the hdf5 file
+        # Create and open the hdf5/n5 file
         export_components = PathComponents(self.ExportPath.value)
         try:
-            os.remove(export_components.externalPath)
+            if export_components.extension in OpStreamingH5N5Reader.N5EXTS:
```

Comment:
Here I see the potential danger, that `OpStreamingH5N5Reader.N5EXT` and `self._export_impls` might get out of sync...